docs/docs/database/plugins.mdx

```diff
@@ -0,0 +1,135 @@
+---
+title: 'Plugins'
```

Comment:
is there a reason for not calling them by their name, extensions?

Reply:
will change